packages/search-field/src/components/Menu.js

```diff
@@ -35,6 +36,12 @@ const renderItem = (item, onClick, highlighted) => (
   </DropdownItemWrapper>
 );
 
+const MenuWrapper = styled(DropdownMenuWrapper)`
+  top: 72px;
```

Comment:
With `72px` a tiny slice of the first validation message appears above the menu. Perhaps we could decrease this size a little. `70px` seems enough.
![captura de tela de 2018-07-06 09-06-31](https://user-images.githubusercontent.com/17071199/42377974-eddf7704-80fb-11e8-9a2c-628474889e54.png)

Reply:
didn't happen on mac... maybe a linux issue (?)
tested on firefox and chrome... but I think it's not a big deal. I'll merge this one (:
![image](https://user-images.githubusercontent.com/17936244/42392533-93ac58fa-8129-11e8-9fe4-cd21ba8014c2.png)